templates/partials/committee/receipts-tab.html

```diff
@@ -6,7 +6,7 @@
   <div class="container">
     <div class="section__heading">
       <h2 class="heading__title" id="section-2-heading">
-        Money received by the <span class="term" data-term="Committee">committee</span>
+        Money received by the committee - {{ cycle|fmt_year_range }}
```

Comment:
Just curious--why drop the glossary lookup here? I'm not clear on when we want to wrap terms in glossary lookups and when we don't.